lib/server.coffee

```diff
@@ -5,6 +5,18 @@ crypto = Npm.require 'crypto'
 
 cacheCollection = new Mongo.Collection 'SpiderableCacheCollection'
 
+hashCode = (string) ->
```

Comment:
Should we add test code to prove that this function really generates unique strings for given urls? What if the URLs are very similar, or have some property that generates the same hashcode?

Should probably add a reference to the algorithm...